### DISCUSSION TITLE
Image with restic 0.17

### DISCUSSION THREAD

**Discussion by jpuchois** [djmaze/resticker#220] [General]
Hello,

Any news about the availability of a tag containing the restic 0.17 release ? 

Thanks :)

**Comment by djmaze** [comment]
Sorry, have been distracted in recent weeks. Will try to update stuff in the coming days.

**Comment by Schwarzarbeit** [comment]
See: https://github.com/djmaze/resticker/discussions/229

Also the new release contains restic version: 1.8.0 https://github.com/djmaze/resticker/releases/tag/1.8.0
